core/tests/test_images.py

```diff
@@ -120,3 +122,34 @@ def test_image_save():
 
     image.save(save_path)
     assert_action_performed_with(image, "save", path=save_path)
+
+def test_pil_support():
+    pil_img = PIL_Image.open("resources/toga.png")
+    toga_img_from_pil_img = toga.Image(pil_img)
```

Comment:
This name was inherited from a previous implementation. It can be significantly simplified now.